src/renderer/Cargo.toml

```diff
@@ -25,6 +25,7 @@ gfx_window_glutin = "0.14"
 glutin = "0.7"
 mopa = "0.2"
 specs = "0.7"
+fnv = "1.0.5"
```

Comment:
Please try to keep the dependencies in alphabetical order. Also, I would recommend omitting the `.5` at the end so Cargo can automatically update patch releases.